packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -95,7 +95,18 @@ module.exports = strapi => {
         schemaDef.resolvers = resolvers;
       }
 
-      const apolloServerConfig = _.get(strapi.plugins.graphql, 'config.apolloServerConfig', {});
+      // TODO: Remove these deprecated options in favor of `apolloServerConfig` in the next major version
+      const defaultApolloServerConfig = {
+        tracing: _.get(strapi.plugins.graphql, 'config.tracing', false),
+        introspection: _.get(strapi.plugins.graphql, 'config.introspection', true),
+        engine: _.get(strapi.plugins.graphql, 'config.engine', false),
+      };
+
+      const apolloServerConfig = _.get(
+        strapi.plugins.graphql,
+        'config.apolloServerConfig',
+        defaultApolloServerConfig
```

Comment:
empty object here so we realy separate the two

Reply:
Done!